### PR TITLE
update prediction functions

### PR DESCRIPTION
**Pull request by double-y**
### Description
prediction functions return both link and response if type.predict is not indicated and the models are glm

### Checklist

Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [x] Add test cases for this fix/enhancement
- [x] Pass devtools::check()
- [x] Pass devtools::test()
- [x] Test installing from github
- [x] Tested with Exploratory

### CODE REVIEW

File: tests/testthat/test_build_glm.R [hunk 1]
```diff
@@ -118,12 +118,25 @@ test_that("prediction with categorical columns", {
...
 
   ret <- prediction(model_data, test_data, type.predict = "response")
+  both_ret <- prediction(model_data, test_data)
+  train_ret <- prediction(model_data, test_data, test = FALSE)
+
   expect_true(nrow(ret) > 0)
   expect_true(all(ret["Fitted"] >= 0 & ret["Fitted"] <= 1))
   expect_equal(colnames(ret), c("CANCELLED", "Carrier.Name", "CARRIER", "DISTANCE", "Fitted", "Standard Error"))
 
+  expect_true(all(both_ret["Fitted.response"] >= 0 & both_ret["Fitted.response"] <= 1))
```
Comment (hideaki) [inline tests/testthat/test_build_glm.R]: Why does prediction adds "Fitted.response" column while add_prediction adds "fitted.response"?

Comment (double-y) [inline tests/testthat/test_build_glm.R]: Fitted.response appears in model info view but fitted.response appears in data frame, so I think lower case is better for it

File: R/broom_wrapper.R [hunk 2]
```diff
@@ -289,6 +341,13 @@ prediction <- function(df, source_data, test = TRUE, ...){
       }
     })
 
+    if (with_response){
+      augmented <- augmented %>%
+        dplyr::ungroup() %>%
+        dplyr::mutate(data = purrr::map2(data, model, add_response)) %>%
+        dplyr::rowwise()
```
Comment (hideaki) [inline R/broom_wrapper.R]: What is this rowwise() for?

Comment (double-y) [inline R/broom_wrapper.R]: Ah, probably not needed

**Pull request merged**